CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ project adheres to [Semantic Versioning](https://semver.org/).
   handles when starting a child process. In particular this means that a serial
   device can be reopened after making SW update of a Tauri application.
   [#130](https://github.com/serialport/serialport-rs/pull/130)
+* Prefers USB device manufacturer and model information from the actual USB
+  device over the information from udev's database. This raises the MSRV from
+  1.56.1 to 1.59.0.
+  [#137](https://github.com/serialport/serialport-rs/pull/137)
```

Comment:
```suggestion
  device over the information from udev's database.
  [#137](https://github.com/serialport/serialport-rs/pull/137)
  * MSRV was raised from 1.56.1 to 1.59.0.
```
I would prefer the MSRV bump being listed on top level.

Reply:
Good point. Looked like burying after reading over it a second time. 😅 Fixed.